Assets/MixedRealityToolkit/SpatialMapping/Scripts/RemoteMapping/FileSurfaceObserver.cs

```diff
@@ -4,7 +4,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-namespace MixedRealityToolkit.SpatialMapping.RemoteMapping
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace HoloToolkit.Unity.SpatialMapping
```

Comment:
Revert the namespace change for the Dev Branch, or retarget May or June Releases